README.md

```diff
@@ -103,7 +97,9 @@ The first line adds a Hadamard gate to circuit object `c` which will operate on
 
 **Note**: Unlike C++ or Python, indexing in Julia starts from "1" and not "0"!
 
-The next step we want to take is to simulate our circuit. We do not need to transpile our circuit since our simulator can handle all gates, but for larger circuit you should consider transpilation to reduce the amount of work the simulator has to perform.
+Once we've built our circuit, we can consider what, if any, transpilations we should apply. Transpilation is the process of rewriting the sequence of operations in a circuit to a new sequence. Typically, the new sequence will yield the same quantum state as the old sequence but could optimize the choice of gates used for performance or portability to different hardware. Since the circuit is relatively small and Snowflurry's simulator can handle all gates, we won't run any transpilation.
```

Comment:
I would change reformulate a bit:
```
Once we've built our circuit, we can consider if it would benefit from applying any transpilation operations. Transpilation is the process of rewriting the sequence of operations in a circuit to a new sequence. As a rule, the new sequence will yield the same quantum state as the old sequence but possibly optimizing the choice of gates used for performance, using only those gates supported by a specific hardware QPU. Since the circuit is relatively small and Snowflurry's simulator can handle all gates, we won't run any transpilation for the time being.
```